va/va.h

```diff
@@ -781,6 +781,18 @@ typedef enum
      * implementation, multiple frames encode/decode can improve HW concurrency
      */
     VAConfigAttribMultipleFrame         = 40,
+    /** \brief indicates the number of scalable pipes minus 1 that the underline encoder supports.
+     * Application may use this number to decide the number of tile columns if scalable pipes are used:
+     * in case of scalable pipeline (number of scalable pipes > 1), number of tile columns should be 
+     * less or equal to (number of scalable pipes minus 1) + 1.
```

Comment:
"number of tile columns should be ".. what about tile rows?

Reply:
You may change the parameter name to something like VAConfigAttribOptMaxTileColumnsMinus1. 
And add some comment like:
When app sets the tile column number equaling to OptMaxTileColumnsMinus1 +1, the performance could be optimized due to parallel processing. But app may also choose other tile column numbers which are not subject to this restriction. There is no such restriction for tile rows.